src/installation/guides/chroot.md

```diff
@@ -0,0 +1,235 @@
+# Installation via chroot (x86 or x86_64)
+
+## Prepare Filesystems
+
+These instructions can be followed on any running system that has xbps installed
+and is able to enter a Linux chroot, including an already working installation
+of Void or an official [live image](../live-images/prep.md). On non-Void
+distributions, there are [statically linked xbps
+binaries](https://alpha.de.repo.voidlinux.org/static/) available for Linux that
+will run without needing any dependencies. It is assumed that the host computer
+is connected to the Internet.
+
+[Partition your disks](../live-images/partitions.md) and format them using
+[mke2fs(8)](https://man.voidlinux.org/mke2fs.8),
+[mkfs.xfs(8)](https://man.voidlinux.org/mkfs.xfs.8),
+[mkfs.btrfs(8)](https://man.voidlinux.org/mkfs.btrfs.8) or whatever tools are
+necessary for your filesystem(s) of choice.
+
+[cfdisk(8)](https://man.voidlinux.org/cfdisk.8) and
+[fdisk(8)](https://man.voidlinux.org/fdisk.8) are available on live
+distributions for partitioning, but you may wish to use
+[gdisk(8)](https://man.voidlinux.org/gdisk.8) (from the package `gptfdisk`) or
+[parted(8)](https://man.voidlinux.org/parted.8) instead.
+
+> Note: For a UEFI booting system, make sure to create a UEFI System Partition
+> (ESP). The ESP should have the partition type "EFI System" (code `ef00`) and
+> be formatted as fat32 using
+> [mkfs.vfat(8)](https://man.voidlinux.org/mkfs.vfat.8).
+
+If unsure what partitions to create, create (1) 1GB in size with type `EFI
+System` (code EF00) (2) remainder of the drive with type `Linux Filesystem`
+(code 8300).
+
+Then format these partitions with fat32 and ext4 respectively.
+
+```
+# mkfs.vfat /dev/sda1
+# mkfs.ext4 /dev/sda2
+```
+
+### Create a New Root and Mount Filesystems
+
+This guide will assume the new root filesystem is mounted on `/mnt`. You may
+wish to mount it elsewhere.
+
+If using UEFI, mount the EFI System Partition as `/mnt/boot/efi`.
+
+For example, if `/dev/sda2` is to be mounted as `/` and `dev/sda1` is the EFI
+System Partition
+
+```
+# mount /dev/sda2 /mnt/
+# mkdir -p /mnt/boot/efi/
+# mount /dev/sda1 /mnt/boot/efi/
+```
+
+Initialize swap space, if desired using
+[mkswap(8)](https://man.voidlinux.org/mkswap.8).
+
+## Base Installation
+
+[Select a mirror to download packages
+from.](../../xbps/repositories/mirror/index.md) For simplicity, save this URL to
+a shell variable for later use. Append `/current` to this repository url to
+install a glibc distribution, append `/current/musl` for a musl distribution.
```

Comment:
"index.md). For"

Reply:
The period is inside the hyperlink, should it be outside?

Reply:
Right now it isn't inside either... Regarding personal preference, I'd put it outside.

Reply:
Just saw it's inside the hyperlink indeed, sorry!

Reply:
> Regarding personal preference, I'd put it outside.

I would normally put the period outside the hyperlink, except that the hyperlink spans the whole sentence. Having the whole sentence hyperlinked might be a bad style choice in and of itself. What do you think about having only "mirror" be the hyperlink?

Reply:
Make the link "Select a mirror" to increase target size and make the target text an action rather than a noun. While you're at it, change "to download packages from" to "from which to download packages" or something like "that will serve packages" to fix the grammar.

Reply:
yeah, that's better

Reply:
actually, I think that second part of the sentence can be left out entirely, I'll just change "mirror" to "XBPS mirror" so that it's clear what the mirror is for. If the reader is unclear, reading the linked document should clarify.

Unfortunately, that brings it back to the whole sentence being a hyperlink.